pkg/upgrade/upgrade_state.go

```diff
@@ -309,6 +327,23 @@ func (m *ClusterUpgradeStateManagerImpl) getPodsOwnedbyDs(ds *appsv1.DaemonSet,
 	return dsPodList
 }
 
+// getOrphanedPods returns a list of the pods not owned by any DaemonSet
+func (m *ClusterUpgradeStateManagerImpl) getOrphanedPods(pods []corev1.Pod) []corev1.Pod {
+	podList := []corev1.Pod{}
+	for i := range pods {
+		pod := &pods[i]
+		if m.isOrphanedPod(pod) {
+			podList = append(podList, *pod)
+		}
+	}
+	m.Log.V(consts.LogLevelInfo).Info("Total orphaned Pods found:", "count", len(podList))
+	return podList
+}
+
+func (m *ClusterUpgradeStateManagerImpl) isOrphanedPod(pod *corev1.Pod) bool {
```

Comment:
minor nit: is there a reason `isOrphanedPod` is a member of `ClusterUpgradeStateManagerImp`?

Reply:
No reason, removed